RELEASE.md

```diff
@@ -91,7 +91,7 @@ The Release Manager must create a release candidate branch:
 -   [ ] 7.  Push the commit
 
     ```bash
-    git push --tag && git push --set-upstream origin 'release-x.x.x'
+    git push --tag origin 'x.x.x-RC-1' && git push --set-upstream origin 'release-x.x.x'
```

Comment:
Interestingly, the correct flag seems to be `--tags` (plural), but `--tag` also works. :thinking:

Reply:
Git has a lot of these aliases